src/topp/CometAdapter.cpp

```diff
@@ -294,8 +294,8 @@ class TOPPCometAdapter :
       {
         OPENMS_LOG_WARN << "Comet v2024.01.0 is known to have several bugs (see https://github.com/UWPR/Comet/issues/63). Please use a different version if possible." << std::endl;
       }
-      // Comet v2024.01.0 introduces �peptide_mass_tolerance_lower� and �peptide_mass_tolerance_upper� parameters
-      // and deprecates �peptide_mass_tolerance� (which is buggy in this version, see https://github.com/UWPR/Comet/issues/59)
+      // Comet v2024.01.0 introduces �peptide_mass_tolerance_lower?and �peptide_mass_tolerance_upper?parameters
```

Comment:
@cbielow these characters should be removed right e.g. they don't have a special meaning in the docs? Some copy and paste artifacts?

Reply:
This may be a character encoding error.

Reply:
indeed.
Just replace with ASCII quotes